data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/beacon/EpochCommitteeResponse.java

```diff
@@ -35,20 +37,24 @@
   public final UInt64 slot;
 
   @JsonProperty("validators")
-  public final List<Integer> validators;
+  @ArraySchema(schema = @Schema(type = "string", example = EXAMPLE_UINT64))
+  public final List<UInt64> validators;
 
   public EpochCommitteeResponse(
       tech.pegasys.teku.datastructures.state.CommitteeAssignment committeeAssignment) {
     this.slot = committeeAssignment.getSlot();
     this.index = committeeAssignment.getCommitteeIndex();
-    this.validators = committeeAssignment.getCommittee();
+    this.validators =
+        committeeAssignment.getCommittee().stream()
+            .map(UInt64::valueOf)
+            .collect(Collectors.toList());
```

Comment:
nit: This is feeling like a common util method we're likely to wind up using in a few places converting from List<Integer> to List<UInt64>.

Reply:
True, I'd be surprised if we don't have this written in a bunch of places...